src/ui/public/embeddable/embeddable_handlers_registry.js

```diff
@@ -0,0 +1,9 @@
+import { uiRegistry } from 'ui/registry/_registry';
```

Comment:
We are already starting to deviate elsewhere - savedObjectsRegistery.  Though yes most are in the ui/registry folder.  If you feel strongly about it I'll move it over to ui/registry, though I'd rather not. :)